### DISCUSSION TITLE
关于 Markdown 的一些奇技淫巧 — 码志

### DISCUSSION THREAD

**Discussion by mzlogin** [mzlogin/blog-comments#215] [Announcements]
https://mazhuang.org/2017/09/01/markdown-odd-skills/

**Comment by tulies** [comment]
哈哈

**Comment by BruceZhaoR** [comment]
同 vim 党，但是最近换到了 vscode，很好用，很方便。

**Comment by mzlogin** [comment]
@BruceZhaoR vscode 是做得不错，界面好看，速度也还行，我也偶尔用。

**Comment by ghost** [comment]
评论功能不错！

**Comment by shawn2016** [comment]
mac我用的是它-typora

**Comment by mzlogin** [comment]
> @shawn2016
> mac我用的是它-typora 

口碑不错~

**Comment by LonlyStack** [comment]
感谢，已经完美复制使用了

**Comment by bjddd192** [comment]
非常感谢你的分享，学习了，👍

**Comment by Fechin** [comment]
再推荐个[在线Markdown生成器](https://tableconvert.com)

**Comment by jhinkoo331** [comment]
我还是比较喜欢..吧vim当插件用23333333333
